src/test/resources/features/DeleteSelectionProcess.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Delete SelectionProcess
+  In order to delete a selection process instance
+  As a user with role secretary
+  I want to delete the a selection process
+
+  Background:
+    Given There is a registered administrator with username "admin" and password "password" and email "admin@sample.app"
+
+  Scenario: User with secretary role delete a selection process
+    Given I login as "admin" with password "password"
+    When I create a selection process with vacancy "sample"
```

Comment:
This is a feature about deletion, thus, the existence of the selection process to be deleted should be part of the Given steps. You just need to change the step type to "And" in this case